### ISSUE TITLE
[ScriptIssue] Inkompatible with osTeleportAgent

### ISSUE THREAD

**Issue by BogusCurry** [WhiteCoreSim/WhiteCore-Dev#292]
Hi ;D
I have load a region with many scripts, when load the region, comes this errors messages
https://paste.md-5.net/lesoweyivo.avrasm

Is that a bug or must I set in the inis ?

Greetings
Bogus

actual git-repo under Ubuntu 16.04 x64 mono 5.0.1.1.

**Comment by greythane** [comment]
Hi Bogus,
This problem is due to the maximum threat level of scripts being set as 'Moderate' by default and your teleport script requires a 'High' threat level.
You will need to adjust the 'FunctionThreatLevel = ...' in Config/Sim/Scripting/DotNetEngine.ini

**Comment by BogusCurry** [comment]
Hello ;D
Ok I have here more script issues, this errors messages comes by start from standalone. 
https://paste.md-5.net/nohigajilu.vhdl

Anyone a idea for a fix ?

Have a nice Day ;D

Greetings
Bogus

**Comment by emperorstarfinder** [comment]
Well that is interesting.  Invalid argument usually means the script has an invalid argument of some sort within the script but usually gives a line number.  The threat level indicates that you have the threat level set higher then is allowed for that script. 

Can you send me the script and ill have a look to see if there is something a miss with DNE.

**Comment by BogusCurry** [comment]
When I found the script, sure. I have load a oar from region, that was in OS online. And the erros from my first post and now this in the second.

**Comment by greythane** [comment]
The bottom line indicates that the function requires a threat level of  veryhigh   but your current settings only allow functions up to a  high  threat level.   You will need to adjust this in your ini   file as previously. 

The invalid argument may be due to some inconsistency between opensim and whitecore which I will check out.

**Comment by emperorstarfinder** [comment]
It actually sounds more like DNE is just being a fussy pain in the butt as normal.  Do you happen to have the link to that OAR?

**Comment by BogusCurry** [comment]
No, is a private oar

**Comment by emperorstarfinder** [comment]
Ahhh Okay.  If it is a newer OAR that was created recently it may be partly due to the changes going on in Opensim.  One being the improper and incorrect changing of ThreatLevels in their scripting engine. If it is an older OAR then it is possible that DNE is just being fussy as normal.

**Comment by BogusCurry** [comment]
Ok ;D The most script issues have I solved ;D But two scripts/items are make erros messages
https://paste.md-5.net/wenalaviba.vhdl

And now I can not found this 2 items, so my question know anyone a objektsearc script so I can delete this bullshit of crap ?

Greetings

Bogus

**Comment by emperorstarfinder** [comment]
If you look in your estate manager you can see the top scripts in the region you might be able to locate them tht way.  Unfortunately I don't have a script on me that can search for them.  Additionally you could attempt to return all objects in your about land (parcel details window)

**Comment by BogusCurry** [comment]
yes, I know this option, but I can not found the both bullet Item ;( 

And return the objects .. oh my over 5000 items .. no is not a option ..

**Comment by emperorstarfinder** [comment]
Well bullets are projectiles and usually are temp rez so I am sort of surprised they didn't get deleted by now.  I will have to see if I can come up with a simple script for you to find them.  Only other option which I am sure is not really an option is the return all objects feature and then re-rez the ones you want out but that is rather time consuming.

**Comment by greythane** [comment]
Check the locations of the appropriate prime...
prim name: Bullet@ <22.53393, 0.2948416, -0.1008495>item name: Bullet
prim name: Bullet@ <23.22695, 1.821792, -0.07556319>item name: Bullet

Also, I have checked the land_collision stuff and there does not appear to be anything unusual here.  The function is expecting a vector3 location as its argument and from previous details, this may be loaded from an notecard (but it does appear this has been corrected)

**Comment by BogusCurry** [comment]
can I found over the console the items and delete the bullets ?

**Comment by emperorstarfinder** [comment]
Hmmm that is actually a good question.  My instinct says yes but i unfortunately don't know that command off hand.

**Comment by greythane** [comment]
Bogus you should be able to locate the prim by transporting in-world nearby to the location of the prim in question.  e.g.: 22.53393, 0.2948416, -0.1008495  Pretty strange locations but should be near bottom left of the region.  If there is land at the normal level (height approx 20) then these objects are below ground... which might explain the land_collision error

You can also use the commands  `show objects` and possibly `show objects Bullet`.
If you can determine the UUID of the objects then they can be deleted using `delete object uuid ` with the uuid you determined.  e.g. `delete object uuid 413c7220-d9e3-11e5-8544-000c297e3c5c`

**Comment by BogusCurry** [comment]
Thanks greythane, but where can found the uuid ?
